src/Components/SingleArticle.jsx

```diff
@@ -8,27 +8,37 @@ import Card from '@mui/material/Card';
 import CardContent from '@mui/material/CardContent';
 import CardMedia from '@mui/material/CardMedia';
 import Typography from '@mui/material/Typography';
+import VoteButton from './VoteButton'
 
 export default function SingleArticle(){
 
   const { articleId } = useParams();
   const [article, setArticle] = useState(null);
   const [loading, setLoading] = useState(true);
   const [comments, setComments] = useState([]);
+  const [totalVotes, setTotalVotes] = useState(0);
 
   useEffect(() => {
     Promise.all([getArticleById(articleId), getCommentsById(articleId)])
     .then(([articleData, commentsData]) => {
       setArticle(articleData);
       setComments(commentsData);
       setLoading(false);
+
+      const commentsVotes = commentsData.reduce((sum, comment) => sum + comment.votes, 0)
```

Comment:
What is the purpose of this code?

Reply:
I wanted my vote count on each article to be the sum of all the votes left in the comments

Reply:
But the article already has it's own 'votes' property that comes back from the api?

Reply:
I thought that too but when I tried to access the votes key it had a value of 0. So i manually added them up. Maybe i am missing it soomewhere

Reply:
the votes would have a 0 as that just means that no one has liked (or disliked) that article yet. The votes on the comments are likes for that particular comment, not a like of the actual article - so if we take the total votes of the comments to be the likes for the article itself, I'm not sure that logic makes sense! You can like a comment some one has left but that doesn't necessary mean you like the article the comment has been left on. Therefore when we vote on the article the vote we should be incrementing is article.votes and not the total of all the comments votes. Hope that makes sense!

Reply:
Okay, I get it. Previously I thought comments were being made with a vote value and the sum of those votes were how votes were cast for an article. But yeah makes more sense now, so I will set votes to article.votes and have it start a zero. Thanks Saima